ecmascript/parser/src/parser/expr.rs

```diff
@@ -776,17 +776,16 @@ impl<'a, I: Tokens> Parser<'a, I> {
         tag: Box<Expr>,
         type_params: Option<TsTypeParamInstantiation>,
     ) -> PResult<'a, TaggedTpl> {
-        let start = tag.span().lo();
+        let tagged_tpl_start = tag.span().lo();
```

Comment:
I'm thinking the key is that we should start giving these more descriptive names to prevent these regressions. `start` is probably too generic.

Reply:
I agree. It's hard to know which position is pointed by `start`.
I'll use some explanatory name in the future.